la.ogri.strongbox.metainfo.xml

```diff
@@ -0,0 +1,39 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="desktop-application">
+    <id>la.ogri.strongbox</id>
+    <name>strongbox</name>
+
+    <keywords>
+        <keyword>warcraft</keyword>
+        <keyword>world of warcraft</keyword>
+        <keyword>addon manager</keyword>
+    </keywords>
+
+    <project_license>AGPL-3.0-only</project_license>
+    <metadata_license>CC0-1.0</metadata_license>
+
+    <summary>strongbox is an addon manager for World of Warcraft.</summary>
+    <description>
+        <p>
+            strongbox is an open source, advertisement free and privacy respecting addon manager for World of Warcraft.
+        </p>
+    </description>
+
+    <launchable type="desktop-id">strongbox.desktop</launchable>
```

Comment:
```suggestion
    <launchable type="desktop-id">la.ogri.strongbox</launchable>
```

Reply:
well actually 
```suggestion
    <launchable type="desktop-id">la.ogri.strongbox.desktop</launchable>
```

but the app-id is still invalid.

Reply:
and the .desktop file is missing.